source/common/runtime/runtime_impl.h

```diff
@@ -43,21 +43,21 @@ class RandomGeneratorImpl : public RandomGenerator {
  * All runtime stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_RUNTIME_STATS(COUNTER, GAUGE)                                                          \
+#define ALL_RUNTIME_STATS(BOOL_INDICATOR, COUNTER, GAUGE)                                          \
   COUNTER(load_error)                                                                              \
   COUNTER(override_dir_not_exists)                                                                 \
   COUNTER(override_dir_exists)                                                                     \
   COUNTER(load_success)                                                                            \
-  COUNTER(deprecated_feature_use)                                                                \
+  COUNTER(deprecated_feature_use)                                                                  \
   GAUGE  (num_keys)                                                                                \
-  GAUGE  (admin_overrides_active)
+  BOOL_INDICATOR  (admin_overrides_active)
```

Comment:
Previously the names of all the stats were aligned. That requires that you re-indent the above 6 lines. Note also that clang-format is disabled for this region.

Reply:
+1 on the nit

Reply:
Done.